pages/03.Howtos/14.add-ptr-records/docs.en.md

```diff
@@ -0,0 +1,145 @@
+
+You may need to install the designate client (plugin).
+
+```shell
+(sudo) pip install python-openstackclient python-designateclient
+```
+
+### Create your zone
+
+We only allow PTR records for zones that you own. You can use a zone that you already created before this how-to.
+
+If you want to create a subzone for the purpose of delegating it to a different project, you may want to work through [this howto](../../02.Tutorials/09.dnsaas/docs.en.md) first.
+
+If you want to practice with a test domain, you can create an empty zone like this:
+
+```shell
+$ openstack zone create --email email@domain.example ptrhowto.example.
++----------------+--------------------------------------+
+| Field          | Value                                |
++----------------+--------------------------------------+
+| action         | CREATE                               |
+| attributes     |                                      |
+| created_at     | 2019-06-28T15:27:00.000000           |
+| description    | None                                 |
+| email          | email@domain.example                 |
+| id             | 01234567-89ab-cdef-0123-456789abcdef |
+| masters        |                                      |
+| name           | ptrhowto.example.                    |
+| pool_id        | 14234f0f-1234-4444-6789-758006f43802 |
+| project_id     | 0123456789abcdef0123456789abcdef     |
+| serial         | 1561735620                           |
+| status         | PENDING                              |
+| transferred_at | None                                 |
+| ttl            | 21600                                |
+| type           | PRIMARY                              |
+| updated_at     | None                                 |
+| version        | 1                                    |
++----------------+--------------------------------------+
+```
+
+### Create a network and a server
+
+You need at least one network and one server. For testing purposes, you can create a network and a server by following the tutorials "[first steps](../../02.Tutorials/01.firststeps/docs.en.md)" or "[single LAMP server](../../02.Tutorials/03.single-lamp-server/docs.en.md)".
+
+### Assign the DNS domain to the network
+
+For automated forward A-record and reverse PTR-record management, the network needs to be assigned to the DNS zone.
+
+Find out the network ID with this command:
+
+```shell
+openstack network list
+```
+
+Then assign the DNS zone to the network:
+
+```shell
+openstack network set --dns-domain ptrhowto.example. <Network UUID>
+```
+
+### Assign the DNS domain and DNS name to the port
+
+Because we did not use the DNS integration when the server was created, we must update the DNS domain and DNS name for the preexisting network port of our server. For newly created servers this will happen automatically, once we assigned the dns domain to the network.
```

Comment:
```suggestion
Because we did not use the DNS integration when the server was created, we must update the DNS domain and DNS name for the preexisting network port of our server. For newly created servers this will happen automatically, once we associated the dns domain with the network.
```